common/testing/historyrequire/history_require.go

```diff
@@ -494,6 +494,13 @@ func (h HistoryRequire) equalExpectedMapToActualAttributes(expectedMap map[strin
 			require.Failf(h.t, "", "Expected property %s.%s wasn't found for EventID=%v", attrPrefix, attrName, eventID)
 		}
 
+		if expectedValue == nil {
+			if !actualV.IsNil() {
+				require.Failf(h.t, "", "Value of property %s.%s for EventID=%v expected to be nil", attrPrefix, attrName, eventID)
+			}
+			continue
+		}
+
```

Comment:
This is a fix in the history event attributes assertion.
If the field is expected to be `nil`, the assertion below at L527 will fail because `actualV` is something like `var actualV any = (*T)(nil)`, and this is different than `var expectedValue any = nil`.